<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>345</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="SeriesGroup" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Line Series</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="pqTreeWidget" name="YAxisArrays" >
        <property name="whatsThis" >
         <string>This displays the list of line series. You can use the list to edit the series options.</string>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Variable</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Color</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>X Axis Data</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QComboBox" name="XAxisArray" >
        <property name="whatsThis" >
         <string>Choose the name of the x-axis data array.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="UseDataArray" >
        <property name="whatsThis" >
         <string>Select this option to use a data array to define the x-axis points.</string>
        </property>
        <property name="text" >
         <string>Use Data Array</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="UseArrayIndex" >
        <property name="whatsThis" >
         <string>Select this option to use the indexes of the y-axis array for the x-axis points.</string>
        </property>
        <property name="text" >
         <string>Use Array Index From Y Axis Data</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="AttributeMode" >
     <item>
      <property name="text" >
       <string>Point Data</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqPointData16.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Cell Data</string>
      </property>
      <property name="icon" >
       <iconset resource="../pqComponents.qrc" >:/pqWidgets/Icons/pqCellData16.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Attribute Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QCheckBox" name="ViewData" >
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Toggle visibility of this dataset's geometry.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="whatsThis" >
      <string>Use this option to show/hide the line series in the chart.</string>
     </property>
     <property name="text" >
      <string>Show Line Series in Chart</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pqComponents.qrc" />
 </resources>
 <connections/>
</ui>
